DependencyInjection/MonologExtension.php

```diff
@@ -938,8 +940,47 @@ private function getHandlerClassByType($handlerType)
             'insightops' => 'Monolog\Handler\InsightOpsHandler',
         ];
 
+        $v2HandlerTypesAdded = [
+            'elasticsearch' => 'Monolog\Handler\ElasticaHandler',
```

Comment:
How should this be handed? `ElasticSearchHandler` -> `ElasticaHandler`

but now we have `ElasticsearchHandler` - what should the key be for this one?

Reply:
Seems good like this to me (elastica != elasticsearch)